schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/model/payment/DefaultPaymentService.java

```diff
@@ -91,21 +98,33 @@ static PaymentService instance() {
         }
     }
 
+    private void queueCancelTask(Transaction tx, Key paymentId) {
+        // If the transaction completes add a task to cancel the shopping cart to cancel the ShoppingCart
+        ApplicationData applicationData = ApplicationData.instance();
+        String cancelTaskDelayInMillisecondsKey = PaymentService.class.getName() + ".CancelTaskDelayInMilliseconds";
+        Integer countdownMillis = applicationData.getPropertyWithDefaultValue(cancelTaskDelayInMillisecondsKey, DEFAULT_CANCEL_TASK_DELAY_IN_MILLISECONDS);
```

Comment:
This is going to throw `ClasCastException` if the value if ever set.  The only "number" supported by ApplicationData is **Long**.  Let's fix it before the merge.

Also, what' I've done so far, is to set the value to the default value if it is not set.  Only because we don't have an interface to edit ApplicationData properties, and when browsing the datastore via console, it's easier if it already exists.